docs/source/workflows/performance_workflows.rst

```diff
@@ -2,12 +2,23 @@ Performance workflows
 =====================
 
 A :class:`performance workflow <citrine.informatics.workflows.PerformanceWorkflow>` performs analysis on a module.
-On construction, a performance workflow requires a configuration object which stores all settings required to run the analysis.
-Currently, the only implemented analysis performs cross-validation on a predictor.
-Settings used to perform cross-validation are defined by a :class:`~citrine.informatics.analysis_configuration.CrossValidationAnalysisConfiguration`.
+Each analysis computes one or more performance metrics, e.g. accuracy of an ML predictor.
+An analysis is codified by a configuration object that stores all relevant settings and parameters required to run the workflow.
+By storing the configuration, the object captures domain-specific knowledge used to characterize a module.
+With this information we know what settings were used in past analyses and can run the same analysis again in the future.
+For example, we might reuse an analysis to compute a specific metric across a range of modules (to determine which module is best suited for our application) or to record the same analysis for a range of different module settings.
+
+On construction, a performance workflow requires an analysis configuration object that stores all settings required to run the analysis.
```

Comment:
This line seems redundant with line 5

Reply:
I think this entire paragraph was left over from an earlier copy-paste. I've revised the intro text and push the changes.